Velaptor/OpenGL/GLWindow.cs

```diff
@@ -482,29 +490,33 @@ private void GLWindow_Render(double time)
         /// <param name="keyboard">The system keyboardInput.</param>
         /// <param name="key">The key that was pushed down.</param>
         /// <param name="arg3">Additional argument from OpenGL.</param>
-        private void GLKeyboardInput_KeyDown(IKeyboard keyboard, Key key, int arg3) => this.keyboardInput.SetState((KeyCode)key, true);
+        private void GLKeyboardInput_KeyDown(IKeyboard keyboard, Key key, int arg3)
+            => this.keyboardReactable.PushNotification(((KeyCode)key, true));
 
         /// <summary>
         /// Invoked when any keyboard input key transitions from the down position to the up position.
         /// </summary>
         /// <param name="keyboard">The system keyboardInput.</param>
         /// <param name="key">The key that was released.</param>
         /// <param name="arg3">Additional argument from OpenGL.</param>
-        private void GLKeyboardInput_KeyUp(IKeyboard keyboard, Key key, int arg3) => this.keyboardInput.SetState((KeyCode)key, false);
+        private void GLKeyboardInput_KeyUp(IKeyboard keyboard, Key key, int arg3)
+            => this.keyboardReactable.PushNotification(((KeyCode)key, false));
 
         /// <summary>
         /// Invoked when any of the mouse buttons are pressed into the down position over the window.
```

Comment:
What does "over the window" mean?  Is it needed?  Can you say, "...when the mouse is in focus in the window?"

Reply:
We need to discuss what is going on here.  I am not seeing what you are talking about here IN the PR.

I went to the code base and found what you were talking about.  There is a reason we cannot see it and I will explain later.